test/helpers/cmd.go

```diff
@@ -205,6 +205,43 @@ func (res *CmdRes) Filter(filter string) (*FilterBuffer, error) {
 	return &FilterBuffer{result}, nil
 }
 
+// FilterLines works like Filter, but applies the JSONPath filter to each line
+// separately and returns returns a FilterBuffer for each line. An error is
+// returned only for the first line which cannot be unmarshalled.
+func (res *CmdRes) FilterLines(filter string) ([]FilterBuffer, error) {
+	parser := jsonpath.New("").AllowMissingKeys(true)
+	err := parser.Parse(filter)
+	if err != nil {
+		return nil, err
+	}
+
+	lines := res.ByLines()
+	results := make([]FilterBuffer, 0, len(lines))
+	for i, line := range lines {
+		if len(line) == 0 {
+			continue
+		}
+
+		var data interface{}
+		result := new(bytes.Buffer)
+		err = json.Unmarshal([]byte(line), &data)
+		if err != nil {
+			if err != nil {
```

Comment:
I think that at this point, `err` is really non-nil :)

Reply:
:sweat_smile: Will fix!